### PR TITLE
CentOS-7 / Centos-7-Cuda / CentOS-8-Stream images removed

### PR DESCRIPTION
**Pull request by DeRuina**
entOS-7 / Centos-7-Cuda / CentOS-8-Stream images are now disabled and therefore removed from the documentations

https://csc-guide-preview.2.rahtiapp.fi/origin/DeRuina-patch-1/cloud/pouta/images/

## Proposed changes

Briefly describe the changes you've made here. Remember to add a link to the [preview page](https://csc-guide-preview.2.rahtiapp.fi/origin/) of your branch.

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes the automated tests. If not, visit [pull requests at Travis CI](https://app.travis-ci.com/github/CSCfi/csc-user-guide/pull_requests) and have a look at the job log.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.2.rahtiapp.fi/origin/) (select your branch from the list).

### CODE REVIEW

File: docs/cloud/pouta/images.md [hunk 1]
```diff
@@ -46,41 +43,24 @@ keep the additional packages at a minimum.
...
-drivers installed, but sometimes we are lagging a bit behind. These images
-are huge, so you should use the normal CentOS-7 image if you are not
-using a GPU-flavor VM.
-
-### CentOS-8-Stream and CentOS-9-Stream
-
-While CentOS-7 is still actively supported, [CentOS-8 has been declared end-of-life in December 2021](https://www.centos.org/centos-linux-eol/).
-The CentOS community is now actively maintaining CentOS-8-Stream and
-CentOS-9-Stream instead. The key differences lay in the relationship between
+The CentOS community is now actively maintaining CentOS-9-Stream. The key differences lay in the relationship between
```
Comment (matteopozza) [inline docs/cloud/pouta/images.md:48]: `The key differences lay in the relationship...` <- this sentence is still referring to our previous documentation, in which we were explaining the differences between CentOS-7 and CentOS-8-stream / CentOS-9-stream
I would suggest to remove the whole sentence (from `The key differences..` till `... new packages and updates, are deployed`. This means that you could just have something like:
```
The CentOS community is now actively maintaining CentOS-9-Stream, which is the upstream branch for RHEL...
```

File: docs/cloud/pouta/images.md [hunk 2]
```diff
@@ -46,41 +43,24 @@ keep the additional packages at a minimum.
...
 CentOS and RHEL, especially in how changes, i.e., new packages and updates,
 are deployed:
 
-* with CentOS-7 RHEL is the upstream branch for CentOS, i.e., changes are first
-deployed for RHEL and then for CentOS
-* with CentOS-8-Stream or CentOS-9-Stream, CentOS is the upstream branch for
+* with CentOS-9-Stream, CentOS is the upstream branch for
 RHEL, i.e., changes are first deployed for CentOS and then for RHEL
 
 The resulting operating system is thus possibly less stable compared to its
```
Comment (matteopozza) [inline docs/cloud/pouta/images.md:55]: `... compared to its previous version` <- also here we are referring to CentOS-7. Given that we are not offering CentOS-7 anymore, I would drop the whole sentence.`

File: docs/cloud/pouta/images.md [hunk 3]
```diff
@@ -46,41 +43,24 @@ keep the additional packages at a minimum.
...
 previous version. The CentOS community [emphasizes](https://blog.centos.org/2020/12/future-is-centos-stream/)
 that the Stream version is nevertheless extremely close in terms of stability
 to the corresponding RHEL version and thus recommends using it as a replacement
-for CentOS-8.
+for CentOS-9.
```
Comment (matteopozza) [inline docs/cloud/pouta/images.md:59]: There is no CentOS-9, there is only CentOS-9-Stream.
This last piece of the sentence is still referring to the change of building CentOS (from after RHEL to before RHEL). By now this is not a change worthwhile mentioning. I would drop the whole `and thus recommends using it as a replacement.`

File: docs/cloud/pouta/images.md [hunk 4]
```diff
@@ -46,41 +43,24 @@ keep the additional packages at a minimum.
...
 
 Note that this is the upstream version of the image, i.e., we do not perform
 any change to the image before making it available on our services. The default
-username is `centos` for CentOS-8-Stream and `cloud-user` for
+username is `cloud-user` for
```
Comment (matteopozza) [inline docs/cloud/pouta/images.md:63]: Given that the whole paragraph is now about CentOS-9-Stream only, mentioning `for CentOS-9-Stream` at the end of this sentence might be unnecessary.

File: docs/cloud/pouta/images.md (hunk 1 above)
Comment (DeRuina) [reply docs/cloud/pouta/images.md]: Fixed

**Comment by DeRuina** [comment]
Changes applied. Can @matteopozza review please?

File: docs/cloud/pouta/images.md [hunk 5]
```diff
@@ -46,42 +43,17 @@ keep the additional packages at a minimum.
...
-deployed for RHEL and then for CentOS
-* with CentOS-8-Stream or CentOS-9-Stream, CentOS is the upstream branch for
-RHEL, i.e., changes are first deployed for CentOS and then for RHEL
-
-The resulting operating system is thus possibly less stable compared to its
-previous version. The CentOS community [emphasizes](https://blog.centos.org/2020/12/future-is-centos-stream/)
+### CentOS-9-Stream
+
+The CentOS community is now actively maintaining CentOS-9-Stream, which is the upstream branch for RHEL.
+
```
Comment (matteopozza) [inline docs/cloud/pouta/images.md:49]: Remove this newline

**Comment by DeRuina** [comment]
Changes applied. Can @matteopozza review please?v

File: docs/cloud/pouta/images.md [hunk 6]
```diff
@@ -46,42 +43,17 @@ keep the additional packages at a minimum.
...
+The CentOS community [emphasizes](https://blog.centos.org/2020/12/future-is-centos-stream/)
```
Comment (matteopozza) [inline docs/cloud/pouta/images.md:49]: The sentence has a `nevertheless` which is difficult to understand without context. You could add before the sentence something like the following: `Given that updates and changes are first tested in CentOS and only subsequently deployed for Red Hat Enterprise Linux (RHEL), the resulting operating system is possibly less stable compared to its previous version, i.e., when RHEL was the upstream for CentOS.`

**Review by matteopozza — APPROVED** [review]

**Pull request merged**